packages/world/mud.config.ts

```diff
@@ -205,6 +205,14 @@ export default defineWorld({
       },
       key: [],
     },
+    RegionMerkleRoot: {
```

Comment:
Should we make this table 3d? ie have a y value, would align with the the energy pool table

Reply:
yea fair, then lets make energy region table 2D too!

Reply:
yea sure!